src/server/helpers.py

```diff
@@ -195,6 +192,29 @@ def accentless_sort(value):
     return sorted(value, key=lambda i: strip_accents(i[1]).lower())
 
 
+def get_file_date_info(file, type):
```

Comment:
Is is possible to get tests for this, or am I pushing my luck? 😆

Reply:
Yeah I’ve been lazy on Python tests. Will see what I can add easily here. And if a huge effort then will wait until #1229 is tackled.

Reply:
Added loads of tests for the new functions in https://github.com/HTTPArchive/almanac.httparchive.org/pull/1508/commits/d6883427afe28c5b7c17a8a33d16c7a602123bda

Reply:
Updated this to test all ebooks that are configured. Yeah!